modules/apps/item-selector/item-selector-taglib/src/main/resources/META-INF/resources/repository_entry_browser/page.jsp

```diff
@@ -647,7 +623,7 @@ SearchContainer<?> searchContainer = new SearchContainer(renderRequest, itemSele
 			editImageURL: '<%= editImageURL.toString() %>',
 		</c:if>
 
-		ffItemSelectorSingleFileUploaderEnabled: <%= FFItemSelectorSingleFileUploaderConfigurationUtil.enabled() %>,
+		itemSelectorSingleFileUploaderEnabled: <%= true %>,
```

Comment:
Can this variable change?
if not. it's really needed?

Reply:
> Can this variable change?

yes, unfortunately on [`item-selector-upload-web`](https://github.com/liferay/liferay-portal/blob/30d870a990b049b4b2bb90249b8e303868292c17/modules/apps/item-selector/item-selector-upload-web/src/main/resources/META-INF/resources/upload.jsp#L63-L81) isn't passed and the itemSelectorSingleFileUploaderEnabled and it is disabled.

I hope isn't needed after [LPS-160919](https://issues.liferay.com/browse/LPS-160919) is done in near future.